pgctl/cli.py

```diff
@@ -15,10 +17,12 @@
 from py._path.local import LocalPath as Path
 
 from .config import Config
+from pgctl.service import Service
 
 PGCTL_DEFAULTS = frozendict({
     'pgdir': 'playground',
     'pgconf': 'conf.yaml',
+    'pghome': Path(os.environ.get('XDG_RUNTIME_DIR') or '~/.run', expanduser=True).join('pgctl').strpath,
```

Comment:
I'd remove the expanduser here, since you do it below.
It means there's less difference between user-configured and default values.